Update fournisseur.facture.class.php

developmentOYR opened the pull request:
IRPF reset when editing the label of a supplier invoice #29049

developmentOYR committed:
Update fournisseur.facture.class.php

IRPF reset when editing the label of a supplier invoice #29049

eldy commented on htdocs/fourn/class/fournisseur.facture.class.php:
```diff
@@ -1243,7 +1243,7 @@ public function update($user = null, $notrigger = 0)
 		$sql .= " close_note=".(isset($this->close_note) ? "'".$this->db->escape($this->close_note)."'" : "null").",";
 		$sql .= " localtax1=".(isset($this->localtax1) ? ((float) $this->localtax1) : "null").",";
-		$sql .= " localtax2=".(isset($this->localtax2) ? ((float) $this->localtax2) : "null").",";
+		$sql .= " localtax2=".(isset($this->total_localtax2) ? ((float) $this->total_localtax2) : "null").",";
```
If we use amount into this->total_locatax2 here, we must also use this->total_locatax1 at the previous line.

developmentOYR replied:
Hello Eldy, I just made the changes you asked me for. I have carried out the appropriate checks to verify that everything is working correctly.

All the best

developmentOYR committed:
Update fournisseur.facture.class.php


Hello Eldy, I just made the changes you asked me for. I have carried out the appropriate checks to verify that everything is working correctly.

All the best

developmentOYR reviewed (commented):
Hello Eldy, I just made the changes you asked me for. I have carried out the appropriate checks to verify that everything is working correctly.

All the best

developmentOYR committed:
Merge branch 'develop' into patch-2

developmentOYR committed:
Merge branch 'develop' into patch-2

eldy committed:
Update fournisseur.facture.class.php

developmentOYR commented:
Dear Eldy:

Could you tell me more or less what version these changes will be in?

sonikf commented:
Hi @developmentOYR,
It will be included in v. 20.0.0
You can close the issue now!
